<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWidget</class>
 <widget class="QWidget" name="CalibrationWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>281</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>172</width>
    <height>81</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>104</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="calibTypeLabel">
       <property name="text">
        <string>Calibration type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="calibFramesLabel">
       <property name="text">
        <string># of frames:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="calibType">
       <item>
        <property name="text">
         <string>Full</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Refine</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="calibFrames">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="calibButton">
     <property name="text">
      <string>Start</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
